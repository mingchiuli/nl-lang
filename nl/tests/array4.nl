a=[1,2,3];
println(a[0]);
a[0]=100;
println(a);
