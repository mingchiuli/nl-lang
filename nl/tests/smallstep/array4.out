1
[100,2,3]

